tests/test_strategy_commercial_producer.py

```diff
@@ -270,3 +270,11 @@ def testing_event_market_cycle(commercial_test3, area_test3):
 def test_commercial_producer_constructor_rejects_invalid_parameters():
     with pytest.raises(ValueError):
         CommercialStrategy(energy_rate=-1)
+
+
+def test_market_maker_strategy_constructor_modifies_global_market_maker_rate():
+    import d3a_interface.constants_limits
+    from d3a.models.strategy.market_maker_strategy import MarketMakerStrategy
+    MarketMakerStrategy(energy_rate=22)
+    assert all(v == 22
+               for v in d3a_interface.constants_limits.GlobalConfig.market_maker_rate.values())
```

Comment:
Can you add a unittest, ensuring `MarketMakerStrategy` must never have any child device? Or route me there, if there is already any such kind of test.

Reply:
Sorry, I do not get what you want to test. Children are held in the Area class, therefore all strategies do not have anything to do with child management. 
In addition to that, same goes for the rest of the strategies. I need to try it out first, but I do not think there is such a check yet on the production code. We have to rely on the user not configuring an area with children and adding a strategy at the same time. If you really think this is important I will add it, but I will open a small story that deals directly with that, because it is a totally different matter from the market maker.

Reply:
Yes please make a separate story